docs/stdlib/sequence.rst

```diff
@@ -89,32 +89,34 @@ Sequences
 
 .. note::
 
-   The sequence to be operated on by the functions above is specified
-   by a ``schema::ScalarType`` object.  If the sequence argument is
-   known ahead of time and does not change, the recommended way to pass
-   it is to use the :eql:op:`introspect` operator:
+    Any sequence that can be called upon by either :eql:func:`sequence_next`
+    or :eql:func:`sequence_rest` functions are specified by a
+    ``schema::ScalarType`` object. If the sequence argument is known ahead of
+    time and does not change, the recommended way to pass it is to use the
+    :eql:op:`introspect` operator:
```

Comment:
I like what you're trying to convey here — that the note applies to both this and the `sequence_next` function. Instead of doing it this way, though, I'd rather just write a note that applies to one and replicate it for the other. This way, if someone searches only for the `sequence_next` function in our documentation and doesn't read on to any other functions, they'll still have the benefit of this information.

The phrasing "Any sequence that can be called upon" seems a bit convoluted though, so I'll suggest a rewrite that can be repurposed as a note for `sequence_next` swapping out references to this function for the other one.

```suggestion
    The sequence to reset is specified using a ``schema::ScalarType`` object.
    If the sequence argument is known ahead of time and does not change, we
    recommended you pass it by using the :eql:op:`introspect` operator.
```

Rewrite for `sequence_next`:

```rest
.. note::

    The sequence to increment is specified using a ``schema::ScalarType``
    object. If the sequence argument is known ahead of time and does not
    change, we recommended you pass it by using the :eql:op:`introspect`
    operator.
```